apps/E2E/src/FocusZone/pages/FocusZonePageObject.ts

```diff
@@ -24,20 +23,6 @@ export const enum GridFocusZoneOption {
   Disable,
 }
 
-export const enum GridButtonSelector {
-  Before = 0,
-  One,
-  Two,
-  Three,
-  Four,
-  Five,
-  Six,
-  Seven,
-  Eight,
-  Nine,
-  After,
-}
-
```

Comment:
What was the reasoning behind removing this enum and replacing it with a number?

Reply:
I think that moving away from selector enums and using WebdriverIO in tests would simplify test code, which is why I removed this enum. 

Because of the number of test components for FocusZone, having an enum might be a better idea in this case. @samuelfreiberg?

Reply:
My concern with this change is that the enum seemed better for type safety - ex. with number I think you could pass in a decimal or a negative number and everything will still compile. Also maybe there are only 10(?) grid buttons, so passing in 11 or 12 would cause some error

Reply:
Re-added an enum for typesafety.